modules/core/src/main/java/org/apache/ignite/internal/hlc/HybridClockImpl.java

```diff
@@ -60,9 +60,9 @@ public HybridClockImpl() {
         this.latestTime = currentTime();
     }
 
-    private static long currentTime() {
-        // TODO https://issues.apache.org/jira/browse/IGNITE-23117 Use currentTimeMillis instead.
-        return systemUTC().instant().toEpochMilli() << LOGICAL_TIME_BITS_SIZE;
+    @TestOnly
```

Comment:
This method can't be TestOnly because it's called from the production code.
Just mock System::currentTimeMillis to return expected value.

Reply:
I removed the annotation and noted why the method has to be declared as a public one.